src/components/playback/playbackmanager.js

```diff
@@ -2343,7 +2333,7 @@ class PlaybackManager {
 
             showPlaybackInfoErrorMessage(self, displayErrorCode);
 
-            return Promise.reject();
+            return Promise.resolve();
```

Comment:
Doesn't this mean that `play` will be resolved rather than rejected?

Reply:
Yes but the `catch` does nothing and the `finally` means the function always resolves overall.

Reply:
I thought that `finally` returns a promise with the same state of error. The original code (before #5402) returned a rejected promise, so it is better to keep this.

Reply:
Yeah you are right about `finally`. I don't know why I can never keep that straight. :sweat_smile: